src/test/scala/uk/gov/hmrc/ui/specs/RALD/RentReviewedSpec.scala

```diff
@@ -47,10 +47,16 @@ class RentReviewedSpec extends BaseSpec with StubPage {
       Landlord.landlord()
       Landlord.landlordNameInput(landlordName = "Bob")
       Landlord.familyMemberRadio()
+      continueButtonClick()
 
       Then("The user selects verbal agreement as there agreement type")
       WhatTypeOfAgreement.TypeOfAgreement()
       WhatTypeOfAgreement.verbalRadio()
+      continueButtonClick()
```

Comment:
Remove this button click as def above clicking on radio button already covers continue button click.